modules/components/pages/inputs/postgres_cdc.adoc

```diff
@@ -374,6 +375,22 @@ Specify the maximum number of tables that are processed in parallel when the ini
 
 *Default*: `1`
 
+
+=== `unchanged_toast_value`
+
+Specify the value to emit when unchanged <<receive-toast-and-deleted-values, TOAST values>> appear in the message stream. Unchanged values occur for data updates and deletes when `REPLICA IDENTITY` is not set to `FULL`.
+
+*Type*: `any`
+
+*Default*: `null`
+
+```yml
+# Examples
+
+unchanged_toast_value: __redpanda_connect_unchanged_toast_value__ 
+# Appears as {"toast":"__redpanda_connect_unchanged_toast_value__"} in a message
```

Comment:
```suggestion

unchanged_toast_value: {"$": "unchanged_toast_value"}
```

Reply:
It appears exactly as it does in the configuration in the message, but reinforcing that `type` is `any` by having two different types for this field.